src/ImageGallery.jsx

```diff
@@ -894,12 +934,12 @@ export default class ImageGallery extends React.Component {
 
     switch (key) {
       case LEFT_ARROW:
-        if (this.canSlideLeft() && !this.intervalId) {
+        if (this.canSlideLeft() && !this.intervalId && isInFocus) {
```

Comment:
Minor performance tip:
Do the `this.canSlideLeft()` check last as this is the only value which seems to require some calculation.
`isInFocus` and `!this.intervalId` are probably quick to determine and if one evaluates to `false`, other (heavier) checks become obsolete and are not performed.
```suggestion
        if (!this.intervalId && isInFocus && this.canSlideLeft()) {
```